standalone/index.js

```diff
@@ -1,114 +1,132 @@
-  APP_RENDER_NOTES: [
-    (store, { latestType, newNoteCount }) =>
-      newNoteCount > 0
-        ? sendMessage({
-            action: 'render',
-            num_new: newNoteCount,
-            latest_type: latestType,
-          })
-        : sendMessage({ action: 'renderAllSeen' }),
-  ],
-  CLOSE_PANEL: [() => sendMessage({ action: 'togglePanel' })],
-  OPEN_LINK: [(store, { href }) => window.open(href, '_blank')],
-  OPEN_SITE: [
-    (store, { siteId, href }) => {
-      sendMessage({ action: 'openSite', siteId });
-      window.open(href, '_blank');
-    },
-  ],
-  OPEN_POST: [
-    (store, { siteId, postId, href }) => {
-      sendMessage({ action: 'openPost', siteId, postId });
-      window.open(href, '_blank');
-    },
-  ],
-  OPEN_COMMENT: [
-    (store, { siteId, postId, commentId, href }) => {
-      sendMessage({ action: 'openComment', siteId, postId, commentId });
-      window.open(href, '_blank');
-    },
-  ],
-  SET_LAYOUT: [
-    (store, { layout }) =>
-      sendMessage({
-        action: 'widescreen',
-        widescreen: layout === 'widescreen',
-      }),
-  ],
-  VIEW_SETTINGS: [() => window.open('https://wordpress.com/me/notifications')],
+    APP_IS_READY: [() => sendMessage({ action: "iFrameReady" })],
+    APP_RENDER_NOTES: [
+        (store, { latestType, newNoteCount }) =>
+            newNoteCount > 0
+                ? sendMessage({
+                      action: "render",
+                      num_new: newNoteCount,
+                      latest_type: latestType
+                  })
+                : sendMessage({ action: "renderAllSeen" })
+    ],
+    CLOSE_PANEL: [() => sendMessage({ action: "togglePanel" })],
+    OPEN_LINK: [(store, { href }) => window.open(href, "_blank")],
+    OPEN_SITE: [
+        (store, { siteId, href }) => {
+            sendMessage({ action: "openSite", siteId });
+            window.open(href, "_blank");
+        }
+    ],
+    OPEN_POST: [
+        (store, { siteId, postId, href }) => {
+            sendMessage({ action: "openPost", siteId, postId });
+            window.open(href, "_blank");
+        }
+    ],
+    OPEN_COMMENT: [
+        (store, { siteId, postId, commentId, href }) => {
+            sendMessage({ action: "openComment", siteId, postId, commentId });
+            window.open(href, "_blank");
+        }
+    ],
+    SET_LAYOUT: [
+        (store, { layout }) =>
+            sendMessage({
+                action: "widescreen",
+                widescreen: layout === "widescreen"
+            })
+    ],
+    VIEW_SETTINGS: [() => window.open("https://wordpress.com/me/notifications")]
 };
 
+const setTracksUser = () =>
+    fetchUser(
+        {
+            fields: "ID,username"
+        },
+        (error, data) => {
+            if (error) {
+                return;
+            }
+            window._tkq = window._tkq || [];
+            window._tkq.push(["identifyUser", data.ID, data.username]);
+        }
+    );
```

Comment:
I'm mixed here. In general I approve of this PR, but I feel like we're just duplicating code needlessly by coupling it in to `rest-client` as well. if we have access to `wpcom` here (which I think in some way we might) I'd just as soon see us code it all here because it's not part-and-parcel of the panel itself.

also, I believe we can use the Promise interface with the `wpcom` object.

```js
wpcom()
	.req
	.get( { path: '/me', apiVersion: '1.1' }, { fields: 'ID,username' } )
	.then( ( { ID, username } ) => ( window._tkq = window._tkq || [], window._tkq.push( [ 'identifyUser', ID, username ] ) ) )
```

Reply:
Ah, I see your point; agreed, I'll keep it all in `standalone/index.js`.